install/re-installation.md

```diff
@@ -156,6 +158,20 @@ BMCs to be set back to DHCP before proceeding.
 
 The process is now done, the NCNs are ready for a new deployment.
 
+<a name="configure_DNS"></a>
```

Comment:
Remove the anchors, use cross-reference links instead in your table of content additions.